data/hai/hai missions.txt

```diff
@@ -1813,169 +1851,170 @@ mission "Nanachi Meeting"
+			choice
+				`	"Wait a minute! This group doesn't react well to strangers. How about I setup a meeting first?"`
+				`	"This is a bad idea. Sorry, but I am backing out."`
+					decline
+			`	The Hai looking at each other and chatter quietly among themselves, then Ambassador Yashili speaks up. "You know this group, and how to make the best impression on them." She pauses to whisper to another Hai, who runs off. The ambassador turns back to you. "I will provide you with a letter with a brief introduction to us, an explanation of why we would like to meet them, and an invitation to establish diplomatic relations."`
+			`	Moments later, the second Hai runs back into the spaceport carrying a sealed envelope and hands it to the Ambassador. She turns to you. "Here it is. Please take this to these reclusive humans for us."`
+				accept
+	on stopover
 		conversation
-			`While looking at the outfits that the Coalition have for sale, the hull repair module catches your eye. It uses a small robot to repair the hull of a ship while in flight. You should return to Hai-home and inform the Hai about this technology.`
-				decline
+			`Looking around the spaceport, you are not sure who to talk to about such a request. Eventually you go to the starport information desk to ask them. After a few minutes they tell you that someone will meet with you, and invite you to wait in the cafeteria.`
+			`	After an hour of watching Remnant eating and gesturing to each other, an androgynous Remnant approaches your table. You try to decide what age they are, but while they do not seem young, neither do they seem particularly old.`
+			branch signlanguage
+				has "Remnant: Learn Sign Follow Up: offered"
+			`	"Greetings, Captain <first>." The stranger chants. "My name is Rayn, and my colleague said you had a request from another faction?"`
+				goto stridercontinue
+			label signlanguage
+			`	"Greetings, Captain <first>." The stranger signs. "My name is Rayn, and my colleague said you had a request from another faction?"`
+			label stridercontinue
+			choice
+				`	"The Hai are hoping to learn from someone."`
+				`	"The Hai need help defending their space."`
+			`	"So the Hai want to improve their military technology? I presume this is in response to their long-standing war against the Unfettered?"`
+			choice
+				`	"Yes, although they don't think of it as such."`
+				`	"It isn't a war, it is a defensive action."`
+			`	Rayn looks amused. "It does not matter what they call it, a war by any name is just as bloody. But that aside, how do they know about us to make such a request?"`
+			choice
+				`	"They asked if I knew anyone who had in-flight repair technology, and I said I knew a group of human researchers."`
```

Comment:
So is the player sort of forced into telling a lie/not the whole truth? This isn't exactly how `<first>` describes the Remnant in either case (at least from my own perspective).

Reply:
The two main statements the player makes about the Remnant to the Hai are:
`	"There is a group of humans far from here known as the Remnant who have ships that can repair themselves."`

`	"No," you explain. "They are a reclusive group unknown to most of humanity. They likely wouldn't take kindly to being visited by aliens."`

While the second might be interpreted as implying they are external to mainstream humanity, but they could be just as much referring to some group hidden somewhere within humanity (because humanity seems to like having hordes of secretive groups, so not really surprising, right?)

Reply:
I see where you're coming from, but I do think saying this directly to the Remnant without mentioning that the Hai now their name and that they're something of a secret still feels like misdirection to me.
It's up to you, though. It does get into subjective interpretation of the text, and you're head of Remnant stuff. If you think it's a non-issue, then you've got the final word.

Reply:
When I first read it, my thoughts went straight to what @W1zrad was thinking – it feels like a misdirection in how it is said.